base/src/main/scala/co/uproot/abandon/Config.scala

```diff
@@ -282,6 +283,20 @@ object SettingsHelper {
     val alias = config.optional("alias") { _.getString(_) }
     AccountSettings(ASTHelper.parseAccountName(name), alias)
   }
+
+  def ensureInputProtection(inputs: Set[String], settings: Settings): Unit = {
+    def filenamesToPaths: Seq[String] => Seq[Path] = _.map(Paths.get(_).normalize)
+    def filenamesToConfRelPaths: Seq[String] => Seq[String] = _.filterNot(_ equals "-").map(settings.getConfigRelativePath)
+
+    val inputFiles = filenamesToPaths(inputs.toSeq)
+    val repOutFiles = filenamesToPaths(settings.reports.flatMap(rep => filenamesToConfRelPaths(rep.outFiles)))
+    val expOutFiles = filenamesToPaths(settings.exports.flatMap(exp => filenamesToConfRelPaths(exp.outFiles)))
```

Comment:
It would be nice to move the logic of getting output paths into the Settings class. That way, if we have more types of output files tomorrow, we can remember to include them when the settings class is modified. Also, peculiarities, such as "-" files, can be confined to the Settings class.